tests/pool/test_pool_async.py

```diff
@@ -872,6 +872,46 @@ def failed(pool):
         assert t1 - t0 < 0.2
 
 
+@pytest.mark.slow
+@pytest.mark.timing
+async def test_async_reconnect_failure(proxy):
```

Comment:
This test is the exact duplication of `test_reconnect_failure()`. It would be better to parametrize the existing one with a `@pytest.mark.parametrize("async_cb", [True, False])` marker and define different callbacks:

```python
if async_cb:
    async def failed(pool):
        assert pool.name == "this-one"
        nonlocal t1
        t1 = time()
else:
    def failed(pool):
        assert pool.name == "this-one"
        nonlocal t1
        t1 = time()
```

Reply:
Ah, nice. I'm far from a pytest expert, so this is nice to learn!

Reply:
Okay, this is done.